package.json

```diff
@@ -37,25 +37,27 @@
   },
   "devDependencies": {
     "@axe-core/react": "^4.10.0",
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.14.0",
     "@playwright/test": "^1.41.2",
     "@singlestore/eslint-plugin-react-hooks-disable-import": "^1.0.2",
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.0.1",
+    "@types/eslint__js": "^8.42.3",
     "@types/node": "^22.9.0",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
-    "@typescript-eslint/eslint-plugin": "^6.14.0",
```

Comment:
These 2 `@typescript-eslint/*` packages prevented the ESLint v9 upgrade in DS. Are they not needed anymore, or have they been replaced with some other packages?

Reply:
Yes, those are now removed as per instructions on the `typescript-eslint` documentation:
https://typescript-eslint.io/getting-started#quickstart